packages/supabase_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.0-dev.2
+
+- **FIX**(supabase_flutter): The session is not restored when the application is started. ([#698](https://github.com/supabase/supabase-flutter/issues/698)).
```

Comment:
This will actually be automatically populated during package release, so no need to add it now.